README.md

```diff
@@ -27,8 +27,20 @@ To learn more about Next.js, take a look at the following resources:
 
 You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
 
-## Deploy on Vercel
+## Deployment
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+Vercel is set up to deploy the `main` branch to production. This is currently published at 
```

Comment:
Is there a bit missing here?

Reply:
@arnodel Thanks yes, I've got a bit more to add, just pushed to test the preview build.